src/clojush/pushgp/genetic_operators.clj

```diff
@@ -26,14 +26,14 @@
   genome of the child."
   [argmap]
   (case (:age-combining-function argmap)
-    :max (fn [p1 p2 g] 
```

Comment:
Just wondering what the reason for removing trailing white space was in this pull request? Not that it's a bad thing, but merge conflicts may be a result if other developers are making pushing changes to some of these other functions.

Reply:
Atom just did it without our consent. It's a good change so we just committed and pushed as a non-issue commit.

Reply:
While I understand that removing the trailing whitespace is a Good Thing (it's the sort of thing I would definitely do), I _wouldn't_ do it in this pull request as I think it really clutters up the PR and makes it a lot harder to see the central changes you're proposing here. If you were going to generate a PR on Lee's repo, I would probably remove this commit first.